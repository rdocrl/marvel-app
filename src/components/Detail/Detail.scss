.detail {
  display: flex;
  gap: 2rem;

  &__info {
    text-align: left;
    color: #fff;
  }
}

@media screen and (max-width: 480px) {
  .detail {
    flex-direction: column;
    gap: 0;

    &__info {
      text-align: center;
    }
  }
}
