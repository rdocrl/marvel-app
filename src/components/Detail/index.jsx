import './Detail.scss';

const Detail = ({ title, description, thumbnail }) => {
  return (
    <div className="detail">
      <img src={thumbnail} alt={title} />
      <div className="detail__info">
        <h2>{title}</h2>
        <p>{description || 'No description available'}</p>
      </div>
    </div>
  );
};

export default Detail;
