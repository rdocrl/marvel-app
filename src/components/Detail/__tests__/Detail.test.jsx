import { render } from '@testing-library/react';
import Detail from '../';

describe('<Detail /> tests', () => {
  describe('Snapshots', () => {
    it('Should render', () => {
      const { asFragment } = render(<Detail />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
