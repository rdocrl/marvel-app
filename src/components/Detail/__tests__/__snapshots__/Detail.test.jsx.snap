// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Detail /> tests Snapshots Should render 1`] = `
<DocumentFragment>
  <div
    class="detail"
  >
    <img />
    <div
      class="detail__info"
    >
      <h2 />
      <p>
        No description available
      </p>
    </div>
  </div>
</DocumentFragment>
`;
